src/test/ui/consts/miri_unleashed/mutable_references.rs

```diff
@@ -28,7 +26,6 @@ static MEH: Meh = Meh {
 };
 
 static OH_YES: &mut i32 = &mut 42;
```

Comment:
same here

Reply:
Add a run-pass test ensuring (at runtime!) that we can read from this static and that the value inside of it is `42`

Reply:
Hmm... but there is no way to mutate that, or do you want me to add a similar test with `static mut` instead.

Reply:
There's no reason to mutate, I just want to make sure we can read the `42`